lib/events/modalReportListingEvents.js

```diff
@@ -1,17 +1,27 @@
 if (Meteor.isClient) {
+	// Implement some system for validation
+	// Choices: Is, Angular forms, validate js ... 
+	// Determine how things will be investigated
 
 	Template.ModalReportListing.events({
 		'click .modalSubmitBtn': function(options) {
 			var options = {
-				prohibited_box: $(".prohibitChoice:checked").val(),
-				offensive_box: $(".offenseChoice:checked").val(),
-				irrelevant_box: $(".irrelevantChoice:checked").val(),
-				false_box: $(".falseChoice:checked").val(),
-				compliance_box: $(".complianceChoice:checked").val(),
+				targetUser: $(".listingItemUserName").text(),
+				riskLevel: 1,
+				reasonBox: {
+					prohibited_box: $(".prohibitChoice:checked").val(),
+					offensive_box: $(".offenseChoice:checked").val(),
```

Comment:
Identifier 'offensive_box' is not in camel case.<br>Mixed double and single quotes.